src/main/java/lotto/view/InputView.java

```diff
@@ -14,7 +17,16 @@ private InputView() {
 
     public static int inputTotalPayment() {
         System.out.println(INPUT_TOTAL_PAYMENT);
-        return Integer.parseInt(scanner.nextLine());
+        try{
+            return Integer.parseInt(scanner.nextLine());
+        }catch (NumberFormatException e){
+            System.out.println(new NotNumberException().getMessage());
+            return inputTotalPayment();
+        }
```

Comment:
사소해 보일 수 있지만 일반적인 자바 컨벤션은 맞춰주는게 좋더라구요 :)
```suggestion
        try {
            return Integer.parseInt(scanner.nextLine());
        } catch (NumberFormatException e) {
            System.out.println(new NotNumberException().getMessage());
            return inputTotalPayment();
        }
```

Reply:
수정하였습니다!